defaults/security-ents.go

```diff
@@ -102,6 +102,7 @@ func securityViewEntitlementEnforce(profile secprofile.Profile) (secprofile.Prof
 		osdefs.SysPtrace,
 		osdefs.SysPersonality,
 		osdefs.SysMadvise,
+		// FIXME: try with: osdefs.SysPrctl,
```

Comment:
did we want to make a tracking issue for this?

Reply:
Yes. opened here: https://github.com/docker/libentitlement/issues/54